### DISCUSSION TITLE
Sunflower land for new players.

### DISCUSSION THREAD

**Discussion by popalexandrudan** [sunflower-land/sunflower-land#2419] [Ideas]
Today my GF started playing the game. She has 0 experience with games in general, but still, there were some things she didn't understand, and i think are not friendly for new players.

1. she had issues adding matic to her already existing metamask account (she already created one a few days ago)
3. she clicked on the fruit quest giver, and didn't understand how to plant berries (this i think should be disabled for low levels)
4. she noticed she had 0 SFL and had no idea what to do next nor what this game is about
5. she clicked on the boat, ended in helios and gut stuck there for a few minutes

From here i started to explain every aspect of the game. Although i don't like mobile games that 'lock' you and force you at the same time to click on certain things, i think the game as it is right now, is not very welcoming to new players, and we need new players. I also think that a better starter guide should be created especially with the launch of referrals.

**Comment by brian-man** [comment]
I would expect new players to read the docs before joining the game.  I think that would address some of your concerns.

> 1. she had issues adding matic to her already existing metamask account (she already created one a few days ago)

This is more of a blockchain issue, not really something I want or expect a game to manage for me.

> 2. she clicked on the fruit quest giver, and didn't understand how to plant berries (this i think should be disabled for low levels)

I don't like the idea of blocking players from anything.  Maybe adding some additional text about when fruit patches are available ("level/expansion/whatever not met") in game makes sense.

Also a quick spin in the docs covers how fruit works and when it unlocks:
https://docs.sunflower-land.com/getting-started/about-sunflower-land?q=fruit
https://docs.sunflower-land.com/player-guides/planting-and-harvesting/fruit

> 3. she noticed she had 0 SFL and had no idea what to do next nor what this game is about

https://docs.sunflower-land.com/getting-started/how-to-play

> 4. she clicked on the boat, ended in helios and gut stuck there for a few minutes

https://docs.sunflower-land.com/player-guides/islands

**Comment by popalexandrudan** [comment]
Who reads docs before playing a game ? I think with this mentality you we are giving up on a lot of new potential players.

**Reply by future2back** [reply]
I share the same perspective, my friend, and I believe your feedback is very important. I've always felt that the game lacks information within the game itself. It seems like the game is designed for those who already know how to play, and I can confirm that even for old players, there are moments when the game can be unclear or confusing, leading to frustration. Indeed, the game needs to be more intuitive.

The game is constantly evolving. Yesterday's documentation is different from today's, and it will be different from tomorrow's. So, in the long run, will players have to read a book before they start playing? Is that appealing?
